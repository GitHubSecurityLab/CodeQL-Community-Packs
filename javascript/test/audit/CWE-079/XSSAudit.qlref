audit/CWE-079/XSSAudit.ql
