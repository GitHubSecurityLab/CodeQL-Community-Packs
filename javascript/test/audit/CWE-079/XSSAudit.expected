| app.jsx:26:52:26:62 | searchQuery | React's dangerouslySetInnerHTML is being used. |
